src/test/java/com/zombiecastlerush/building/InventoryTest.java

```diff
@@ -75,4 +76,81 @@ public void transferItem() {
         inventory.transferItem(this.inventory, invTo, item);
         assertTrue(invTo.getItems().contains(item));
     }
-}
+
+    @Test
+    public void testMoveItem_targetItemInDestinationInventoryAfterMovement(){
+        Player player = new Player("Player");
+        Room room = new Room("Room", "just a room");
+        //TODO: need to test Puzzle inventory once updated version available
+        Item item = new Item("target item", "I will be moved between objects");
+        player.getInventory().addItems(item);
+        Assert.assertEquals(room.getInventory().getItems().size(), 0); //empty inventory
+        Assert.assertEquals(player.getInventory().getItems().get(0).getName(),item.getName()); //one item in inventory
```

Comment:
Anywhere you compare the name, you can simply just do...

```
assertEquals(player.getInventory().getItems().get(0), item);
```

The item that you created and added to player inventory using `addItems`, should be one and the same.

Reply:
will try that in future branch